src/tests/Register.test.js

```diff
@@ -1,202 +1,321 @@
-import React from 'react';
-import { rest } from 'msw';
-import { setupServer } from 'msw/node';
-import { render, fireEvent, screen, wait, waitForElement } from '@testing-library/react';
-import '@testing-library/jest-dom';
-import { act } from 'react-dom/test-utils';
+import React from "react";
+import { rest } from "msw";
+import { setupServer } from "msw/node";
+import {
+  render,
```

Comment:
something odd happened with the formatting in this file as well. this negatively affects readability

Reply:
I guess it's because prettier is not working properly for me. I guess the code will be readable after running the prettier. I tried hard to correct this even done manually but after saving everything goes back to the same.So it's quite difficult to format tests.

Reply:
I see. do you have prettier extension or any other linter enabled? it is annoying to do it manually but if you do that and have the prettier disabled it shouldn't autoformat when you save